### PR TITLE
Feature/new job posting

### PR DESCRIPTION
**Pull request by dmytro-chushko**
new-job-posting-api has added
modified new-job-posting-slice
logic for create job button has added
logic for loader of create job button has added
options for SelectElements has fixed
SelectElements has modified

### CODE REVIEW

File: apps/freelance/src/utils/select-options/options.ts [hunk 1]
```diff
@@ -520,23 +524,23 @@ export const workExperience: SelectOptions[] = [
...
-		label: "more than 20 years",
+		label: "More than 20 years",
 		value: "more20",
 	},
 	{
-		label: "more than 25 years",
+		label: "More than 25 years",
 		value: "more25",
 	},
 ];
```
Comment (lyudad) [inline apps/freelance/src/utils/select-options/options.ts]: I dont know what is it. but all texts should be in translation file

File: libs/components/src/lib/job-posting-second-form/job-posting-second-form.tsx [hunk 2]
```diff
@@ -60,16 +60,16 @@ export function JobPostingSecondForm({ page }: IJobPostingFormProps) {
...
 		<JobPostingGridForm id={page} onSubmit={handleSubmit(onSubmit)} justifyItems="center">
 			<IncreasedFieldWrapper gridRow={1} typeOfLength="full">
 				<JobPostingLabel>{countreisLabel}</JobPostingLabel>
 				<ErrorsHandlerWrapper positionRight={-20} width={15}>
 					<Controller
 						name="countries"
-						defaultValue={JSON.parse(JSON.stringify(array))}
+						defaultValue={JSON.parse(
+							JSON.stringify(selectDefaultArray(storedCountries, countries)),
+						)}
```
Comment (lyudad) [inline libs/components/src/lib/job-posting-second-form/job-posting-second-form.tsx]: I dont know what is it, but no, find other way

File: libs/components/src/lib/job-posting-third-form/job-posting-third-form.tsx [hunk 3]
```diff
@@ -50,7 +54,7 @@ export function JobPostingThirdForm({ page }: IJobPostingFormProps) {
 				<ErrorsHandlerWrapper positionRight={-20} width={15}>
 					<Controller
 						name="skills"
-						defaultValue={JSON.parse(JSON.stringify(array))}
+						defaultValue={JSON.parse(JSON.stringify(selectDefaultArray(storedSkills, skills)))}
```
Comment (lyudad) [inline libs/components/src/lib/job-posting-third-form/job-posting-third-form.tsx]: no JSON.parse(JSON.stringify, find other way

File: apps/freelance/src/utils/select-options/options.ts (hunk 1 above)
Comment (dmytro-chushko) [reply apps/freelance/src/utils/select-options/options.ts]: added  to translation file

File: libs/components/src/lib/job-posting-second-form/job-posting-second-form.tsx (hunk 2 above)
Comment (dmytro-chushko) [reply libs/components/src/lib/job-posting-second-form/job-posting-second-form.tsx]: removed JSON.parse, changed logic

File: libs/components/src/lib/job-posting-third-form/job-posting-third-form.tsx (hunk 3 above)
Comment (dmytro-chushko) [reply libs/components/src/lib/job-posting-third-form/job-posting-third-form.tsx]: removed JSON.parse, changed logic

**Pull request merged**